Pilot/pilotTools.py

```diff
@@ -271,24 +271,14 @@ def getSubmitterInfo(ceName):
         flavour = "SSH%s" % batchSystemType
         pilotReference = "sshslurm://" + ceName + "/" + batchSystemJobID
 
-    # Condor
-    if "CONDOR_JOBID" in os.environ:
-        batchSystemType = "HTCondor"
-        batchSystemJobID = os.environ["CONDOR_JOBID"]
-        batchSystemParameters["InfoPath"] = os.environ.get("_CONDOR_JOB_AD", "Unknown")
-
-        flavour = "SSH%s" % batchSystemType
-        pilotReference = "sshcondor://" + ceName + "/" + batchSystemJobID
-
-    # # CEs/Batch Systems
-
     # HTCondor
-    if "HTCONDOR_JOBID" in os.environ:
+    if "_CONDOR_JOB_AD" in os.environ:
         batchSystemType = "HTCondor"
```

Comment:
I think at this point it should be:

```
    # Condor
    if "_CONDOR_JOB_AD" in os.environ:
        batchSystemType = "Condor"
```

Reply:
Why? 
`Condor` is the old name for `HTCondor`: https://lists.cs.wisc.edu/archive/htcondor-users/2012-October/msg00110.shtml
I think it should be the opposite then, we should rename `Condor` as `HTCondor` in DIRAC.